packages/fast-components-styles-msft/src/toggle/index.ts

```diff
@@ -10,9 +10,8 @@ const styles: ComponentStyles<IToggleClassNameContract, IDesignSystem> = {
         "& span": {
             userSelect: "none",
             marginTop: "0",
-            paddingBottom: "0"
-        },
-        "&[aria-disabled='true']": {
+            paddingBottom: "0" },
+        "&[aria-disabled=\"true\"]": {

```

Comment:
Interesting - the single quotes were causing the issue?

Reply:
yep :/